feature_layers/create-mobile-geodatabase/src/main/java/com/esri/samples/create_mobile_geodatabase/CreateMobileGeodatabaseController.java

```diff
@@ -0,0 +1,336 @@
+    geodatabase.close();
+    inputs.clear();
+    tableStage.close();
+
+    // display geodatabase file location
+    Alert dialog = new Alert(Alert.AlertType.INFORMATION,
+      "Mobile geodatabase has been closed and saved in the following directory: " + geodatabasePath.toString());
+    dialog.initOwner(mapView.getScene().getWindow());
+    dialog.setHeaderText(null);
+    dialog.setTitle(("Information"));
+    dialog.showAndWait();
+
+    // handle UI
+    viewTableButton.setDisable(true);
+    closeGeodatabaseButton.setDisable(true);
+    createGeodatabaseButton.setDisable(false);
+    label.setText("");
+    graphicsOverlay.getGraphics().clear();
+    map.getOperationalLayers().clear();
+  }
+
+  /**
+   * Displays a new window with the table of features stored in the geodatabase feature table.
+   */
+  @FXML
+  private void handleDisplayTable() {
+
+    // close previous table
+    handleTableWindowVisibility();
+
+    // create observable list of type GeoFeature to store the geodatabase features
+    final ObservableList<FeatureAttributeField> fieldData = FXCollections.observableArrayList();
+
+    // query all the features loaded to the table
+    var queryResultFuture = geodatabaseFeatureTable.queryFeaturesAsync(new QueryParameters());
+    queryResultFuture.addDoneListener(() -> {
+      try {
+        var queryResults = queryResultFuture.get();
+        queryResults.forEach(feature ->
+          // add features to the observable list
+          fieldData.add(new FeatureAttributeField(feature.getAttributes().get("oid").toString(), feature.getAttributes().get("collection_timestamp").toString())));
+
+        // create and set up a new table view to display the features in a table
+        TableView<FeatureAttributeField> table = new TableView<>();
+
+        // create two table columns and add them to the table view
+        TableColumn<FeatureAttributeField, String> oidCol = new TableColumn<>("OID");
+        TableColumn<FeatureAttributeField, String> timeCol = new TableColumn<>("COLLECTION TIMESTAMP");
+        table.getColumns().add(oidCol);
+        table.getColumns().add(timeCol);
+
+        // associate data to the table columns referencing the fields in the GeoFeature class
+        oidCol.setCellValueFactory(new PropertyValueFactory<>("oid"));
+        timeCol.setCellValueFactory(new PropertyValueFactory<>("timestamp"));
+
+        // add data to the table view
+        table.setItems(fieldData);
+
+        // create a StackPane, Scene, and Stage for displaying the table view in a new window
+        var pane = new StackPane();
+        pane.getChildren().add(table);
+        var scene = new Scene(pane, 220, 230);
+
+        // set up stage properties before display
+        tableStage = new Stage();
+        tableStage.setTitle("Features");
+        tableStage.setScene(scene);
+        tableStage.show();
+
+      } catch (InterruptedException | ExecutionException e) {
+        throw new RuntimeException(e);
```

Comment:
If you are throwing an exception in the method, you should always add this detail to the method signature and doc i.e.:
`private void handleDisplayTable() throws Exception` and 
```
  /**
   * Displays a new window with the table of features stored in the geodatabase feature table.
   * @throws RuntimeException
   */
```

That said, I would change this for the sake of samples to handle the Exception. Other Samples generally display an alert - as you've done elsewhere in this - so I'd follow that pattern and instead do something like:
```
var alert = new Alert(Alert.AlertType.ERROR, "Failed to query the feature table.");
alert.show();
```

There is another instance of this to tidy up too. Give me a shout if you need any more info on this.